packages/nimble-blazor/NimbleBlazor/MappingKeyType.cs

```diff
@@ -0,0 +1,17 @@
+namespace NimbleBlazor;
+
+#pragma warning disable CA1720
```

Comment:
I'm not sure if I'll get pushback on this. The rule disallows type names, including `String` and `Boolean` in identifiers. These names represent JS types, and it would be weird to call them something else.

Reply:
I'll let someone else, like @atmgrifter00, that is more aware of the problems that could arise from these names weigh in, but a possible alternative (if deemed necessary) could be `StringKey`.

Reply:
As I indicated in another comment, if I have my way, this type goes away.

Reply:
Yep, this type went away.